charts/seed-bootstrap/charts/elastic-kibana-curator/templates/searchguard/searchguard-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.searchguard.enabled }}
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: sgadmin
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: logging
+    app: sgadmin
+    role: logging
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: sgadmin
+      role: logging
+  template:
+    metadata:
+      annotations:
+        checksum/searchguard-configmap: {{ include (print $.Template.BasePath "/searchguard/searchguard-configmap.yaml") . | sha256sum }}
+        checksum/searchguard-update-indices-config: {{ include (print $.Template.BasePath "/searchguard/searchguard-update-indices-config.yaml") . | sha256sum }}
+      labels:
+        garden.sapcloud.io/role: logging
```

Comment:
What is the purpose of this label ?

Reply:
I have checked:
https://github.com/gardener/gardener/blob/3e14c96996eab212dd4a4d380d7cc34287e4671f/pkg/operation/common/types.go#L104